public/apps/configuration/utils/password-edit-panel.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { EuiFieldText, EuiIcon } from '@elastic/eui';
+import { FormRow } from './form-row';
+
+// At least one uppercase, one lowercase, one digit and one special character
+// and at least 8 characters long
+const PASSWORD_PATTERN = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?([^\w\s]|[_])).{8,}$/;
```

Comment:
Currently we do not have unit test setup. But to help remember the places which definitely need unit test, probably add a todo here for future prioritization.